src/pqm/__init__.py

```diff
@@ -1,6 +1,5 @@
 from .pqm import pqm_pvalue, pqm_chi2
 from .test_gaussian import test
-from ._version import __version__
```

Comment:
I think we need this line for the automatic version control in pip. Why is it being removed?